assets/css/sensei-course-theme/course-settings-editor.scss

```diff
@@ -0,0 +1,27 @@
+.sensei-course-theme-toggle {
+	margin-bottom: 20px;
+
+	.components-base-control__field {
+		align-items: flex-start;
+	}
+
+	&__label {
+		margin-bottom: 2px;
+	}
+
+	&__description {
+		color: #757575;
+		margin-bottom: 6px;
+	}
+
+	&__preview {
+		margin-bottom: 0;
+	}
+
+	&__note {
```

Comment:
I also didn't see it in use. Is it something we should remove or I missed something?

Reply:
Removed. 👍